chatbot-core/data/chunking/chunking_utils/logger.py

```diff
@@ -0,0 +1,15 @@
+"""Logger configuration for chunking scripts."""
+
+import logging
+
+logging.basicConfig(
+    level=logging.INFO,
+    format='%(asctime)s [CHUNKING] [%(levelname)s] [%(filename)s]: %(message)s',
```

Comment:
Seems we can change [CHUNKING] with 	
%(module)s. Please let me know if it won't help.